spec/models/team_spec.rb

```diff
@@ -147,6 +147,80 @@
     end
   end
 
+  describe 'limit of coaches' do
+    let(:team) { create :team }
+    let(:new_user) { create :user }
+    let(:coach_attributes) { [{ name: 'coach', team_id: team.id, user_id: new_user.id }] + 3.times.map { { name: 'coach', team_id: team.id, user_id: create(:user).id } } }
```

Comment:
This line is a bit long now. Can you make it a multi-line `do … end` let? Also see below for a suggestion on the 3.times.map(&block)

Reply:
Thanks for the suggestion! I've split it up, but can't use create_list for a set of attributes, can I?

Reply:
No, I don't think you can